projects/components/src/lib/togglebutton/toggle-button.component.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  EventEmitter,
+  Input,
+  OnInit,
+  Output,
+  ViewChild,
+  forwardRef,
+} from '@angular/core';
+import { ControlValueAccessor, FormControl, NG_VALUE_ACCESSOR } from '@angular/forms';
+import '@masoni/css-toggle-button/dist/index.css';
+
+export interface ToggleButtonEvent {}
+@Component({
+  selector: 'mas-toggle-button',
+  templateUrl: 'toggle-button.component.html',
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => MasToggleButton),
+      multi: true,
+    },
+  ],
+})
+export class MasToggleButton implements OnInit, ControlValueAccessor {
+  protected _disabled: boolean = false;
+  protected _checked: boolean = false;
+
+  @Input()
+  set checked(value: boolean) {
+    if (this.checked !== value) {
+      this.checked = value;
+    }
+    this.cd.markForCheck();
+  }
+  get checked(): boolean {
+    return this._checked;
```

Comment:
setter did not update value for this._checked

Reply:
issue fixed